src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -15,9 +17,10 @@ import {getChannelListInfo, getChannelMembers} from '../../../utils/string/Strin
 type ChannelCategory = 'SIGNED' | 'ANONYMOUS';
 
 const useFetchChannelHook = () => {
-  const {localDb, refresh} = useLocalDatabaseHook();
+  const {localDb, refresh, refreshWithId} = useLocalDatabaseHook();
   const {signedProfileId, anonProfileId} = useUserAuthHook();
   const {getFirstMessage} = useSystemMessage();
+  const {queue} = useDatabaseQueueHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of `refreshWithId` and `queue` from the hooks seems to be a good addition for better control over data refreshes and efficient management of database operations. However, it would be beneficial to ensure that these new methods are thoroughly tested, especially in edge cases, to avoid potential issues.



<!-- This is an auto-generated comment by OSS CodeRabbit -->